src/main/resources/static/js/ltiPostMessage.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * A class for handling LTI post messages.
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+class LTIPostMessage {
+  /**
+   * Constructs an instance of LTIPostMessage.
+   */
+  constructor(target) {
+    /**
+     * The message ID.
+     * @type {string|null}
+     */
+    this.mId = null;
+
+    /**
+     * The subject of the message.
+     * @type {string|null}
+     */
+    this.subject = null;
+
+    /**
+     * The error message.
+     * @type {string|null}
+     */
+    this.error = null;
+
+    /**
+     * The response message.
+     * @type {object|null}
+     */
+    this.response = null;
+
+    this.target = target;
+
+    /**
+     * Binds the handlePostMessages method to the current instance.
+     */
+    this.handlePostMessages = this.handlePostMessages.bind(this);
+
+    // Setup the event listener
+    window.addEventListener('message', this.handlePostMessages);
+  }
+
+  async create_UUID() {
```

Comment:
Would it be possible for us to import a uuid library like uuidv4 to do this instead of manually generating the uuid?